### ISSUE TITLE
Using file restore with other thermostats (not the programmable one from you)

### ISSUE THREAD

**Issue by jarlesb** [custom-components/sensor.file_restore#1]
Hi, 
and thanks for developing a very good module to be used with the programmable thermostat!

I have implemented it with a couple of heaters I have, but would also like to use it with a floor thermostats (HeatIt floor heating and Mitisubishi Heat pump) that do not use the generic/programmable thermostat module. 

Is it possible to do that? With the code below I only get wrong type, not float....

E.g.:
automation:
- alias: Set temperature heat pump
  trigger:
  - platform: time_pattern
    minutes: "/5"
  action:
  - service: climate.set_temperature
    entity_id: climate.melcloud_XXXX
    data:
      temperature: sensor.set_temperature_heatpump (file_restore sensor)
      hvac_mode: heat

**Comment by MapoDan** [comment]
Ciao Jarlesb,
I think that what you want to do is not possible.
The issue is probably related to fact that the climate object expect to receive a value and not an entity. This is why I had dev my thermostat :)
Maybe something similar can be dev starting from the integration you are using right now.

One Idea to try with an automation as you did is to use template.
if the trigger is working (the / sound strange to me) try to use this data in the action:
``` yaml
data_template:
  temperature: {{ states("sensor.set_temperature_heatpump")|float }}
  hvac_mode: heat
```
Let me know ;)

**Comment by jarlesb** [comment]
Ciao and thanks a lot for reply! 

The trigger every 5 minutes seems to work well with "/". See the Home assistant trigger page for documentation. 

I tried with the template, but get the error:
invalid key: "OrderedDict([('states("sensor.set_temperature_varmepumpe")|float', None)])"
  in "/config/automations.yaml", line 138

I have never used templates before so should probably read up on that... What I did was to replace the code from data in the automation with the one you proposed... But may have missed something. But would be great if it would be possible to get the state out as a float and then use this to update every 5 min (everyb5 min instead of every hour since I have also implemented away_mode and open window...

Will let you know if I manage!
